benchmarks/heart_disease/mindsdb_acc.py

```diff
@@ -0,0 +1,32 @@
+import mindsdb
+import pandas as pd
+import numpy as np
```

Comment:
small thing: np, math, and r2_score imported but never used, so better to remove